galaxy_ng/app/api/urls.py

```diff
@@ -1,16 +1,43 @@
+from django.conf import settings
 from django.urls import include, path
 
 from . import views
 from .ui import urls as ui_urls
 from .v3 import urls as v3_urls
+from .v3 import viewsets as v3_viewsets
 
+DEFAULT_DISTRIBUTION_BASE_PATH = settings.GALAXY_API_DEFAULT_DISTRIBUTION_BASE_PATH.strip('/')
 
 app_name = "api"
 urlpatterns = [
-    path("", views.ApiRootView.as_view(), name="root"),
-    path("v3/", include(v3_urls)),
     path("v3/_ui/", include(ui_urls)),
 
+    # A set of galaxy v3 API urls per Distribution.base_path
+    # which is essentially per Repository
+    path("content/<str:path>/v3/",
+         include(v3_urls, namespace='per_distribution')),
```

Comment:
Think `namespace` should be `name`? Also, `per_distribution` should be `per-distribution` to be consistent with subsequent names.

Reply:
Updated to per-distribution.

include() wants a 'namespace' arg and path() wants a 'name' arg if I understand correctly